.gitignore

```diff
@@ -1,3 +1,82 @@
+.DS_Store
+.vscode
+libsvmdata.egg-info
 
+# Python's gitignore
+__pycache__/
 *.pyc
-libsvmdata.egg-info/*
+*.py[cod]
```

Comment:
that's too much for me, do you need all of these? Otherwise I keep the gitignore minimal